### PR TITLE
Feature/bootcamp fragments

### PR DESCRIPTION
**Pull request by raynasr**
Requesting a review for the Fragments feature.

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+Bootcamp
```
Comment (NervzZ) [inline .idea/.name:1]: From this change I see that you did not rebase master. Please go on your branch (bootcamp-fragments) and do git rebase master then push to update this PR. Then I'll review the rest of the PR.

**Review by NervzZ — CHANGES_REQUESTED** [review]
Changes requested in first comment.

File: .idea/.name (hunk 1 above)
Comment (raynasr) [reply .idea/.name:1]: Thank you, it had slipped my attention.

File: app/src/main/java/com/studhub/app/GreetActivity.java [hunk 2]
```diff
@@ -15,7 +17,15 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Intent intent = getIntent();
         String name = intent.getStringExtra("name");
-        String message = String.format("Hello my dear %s, how is you?", name);
+        String message = String.format("Hello my dear %s, how are you?", name);
```
Comment (edouardmichelin) [inline app/src/main/java/com/studhub/app/GreetActivity.java:20]: intentionnel

**Review by edouardmichelin — CHANGES_REQUESTED** [review]

**Review by NervzZ — APPROVED** [review]
Outside of what Edouard said it looks ok to me. I may ask you a few questions about fragments for my own understanding of the code itself though.

**Pull request closed without merging**